.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "examples/nbody"]
 	path = examples/nbody
-	url = git@github.com:ut-parla/barnes-hut.git
+	url = https://github.com/ut-parla/barnes-hut.git
 [submodule "examples/blr"]
 	path = examples/blr
-	url = git@github.com:ut-parla/BLR.git
```

Comment:
Just curious. Why is this change required?
Technically, both look fine and private repos enforces to use git@github.com format.